ambry-protocol/src/main/java/com.github.ambry.protocol/GetRequest.java

```diff
@@ -40,6 +40,7 @@
   private static final int Partition_Request_Info_List_Size = 4;
   private static final short Get_Request_Version_V2 = 2;
   public static final String Replication_Client_Id_Prefix = "replication-fetch-";
+  public static final String Cloud_Replica_Keyword = "vcr";
```

Comment:
Does this need to be in this class?  Seems like it's only used in cloud package.

Reply:
It's also used in `getRemoteReplicaInfo`, otherwise it reports error: ```ReplicaMetaDataRequest from unknown Replica {}```.